src/pages/console/formbuilder/lifecycle.mdx

```diff
@@ -0,0 +1,163 @@
+export const meta = {
+  title: "Manage form lifecycle",
+  description: "Hook into the form's lifecycle events to customize user input before submission, run validations, handle errors, or self-manage user input events."
+}
+
+Hook into the form's lifecycle events to customize user input before submission, run validations, handle errors, or self-manage user input events. 
```

Comment:
`customize user input before submission` and `self-manage user input events` feel a bit redundant